pycm/pycm_multilabel_cm.py

```diff
@@ -0,0 +1,183 @@
+# -*- coding: utf-8 -*-
+"""MultiLabelCM module."""
+from __future__ import division
+from .pycm_error import pycmVectorError, pycmMatrixError
+from .pycm_param import *
+from .pycm_obj import ConfusionMatrix
+import numpy
+
+
+class MultiLabelCM():
+    """
+    Multilabel confusion matrix class.
+
+    >>> mlcm = MultiLabelCM([[0, 1], [1, 1]], [[1, 0], [1, 0]])
+    >>> #TODO: example
+    >>> mlcm = MultiLabelCM([{'dog'}, {'cat', 'dog'}], [{'cat'}, {'cat'}], samplewise=True)
+    >>> mlcm.cms
+    [pycm.ConfusionMatrix(classes: [0, 1]), pycm.ConfusionMatrix(classes: [0, 1])]
+    >>> mlcm.actual_vector
+    [[0, 1], [1, 1]]
+    >>> mlcm.predict_vector
+    [[1, 0], [1, 0]]
+    """
+
+    def __init__(
+            self,
+            actual_vector,
+            predict_vector,
+            samplewise=False,
+            sample_weight=None,
+            classes=None):
+        """
+        Init method.
+
+        :param actual_vector: actual vector
+        :type actual_vector: python list or numpy array of list or set
+        :param predict_vector: vector of predictions
+        :type predict_vector: python list or numpy array of list or set
+        :param samplewise: flag to calculate confusion matrices per sample
+        :type samplewise: bool
+        :param sample_weight: sample weights list
+        :type sample_weight: list
+        :param classes: ordered labels of classes
+        :type classes: list
+        """
+        self.actual_vector = actual_vector
+        self.predict_vector = predict_vector
+        self.weights = None
+        self.classes = None
+        __mlcm_vector_handler__(
+            self,
+            actual_vector,
+            predict_vector,
+            sample_weight,
+            classes)
+        __mlcm_assign_classes__(self, actual_vector, predict_vector, classes)
+        __mlcm_vector_filter__(self)
+        if samplewise:
+            self.cms = []
+            for actual, predict in zip(
+                    self.actual_vector, self.predict_vector):
+                print(actual, predict)
```

Comment:
Is it for a test?

Reply:
Yes. I thought I removed it but it seems I made a mistake.